custom_components/midea_ac_lan/fan.py

```diff
@@ -48,21 +54,9 @@ async def async_setup_entry(
 
 
 class MideaFan(MideaEntity, FanEntity):
-    def __init__(self, device: Any, entity_key: str) -> None:
+    def __init__(self, device: MideaDevice, entity_key: str) -> None:
         super().__init__(device, entity_key)
 
-    def turn_on(
```

Comment:
Why have you migrated this code?

Reply:
I moved it to subclass, since many subclasses ovrewrite it again.

Reply:
We can go this way for now, but later we need to review all duplicated code and cleanup.